cdci_data_analysis/pytest_fixtures.py

```diff
@@ -393,16 +395,16 @@ def dispatcher_test_conf_with_gallery_fn(dispatcher_test_conf_fn):
 
 @pytest.fixture
 def dispatcher_test_conf_with_renku_options_fn(dispatcher_test_conf_fn):
-    fn = "test-dispatcher-conf-with-gallery.yaml"
+    fn = "test-dispatcher-conf-with-renku-options.yaml"
 
     with open(fn, "w") as f:
         with open(dispatcher_test_conf_fn) as f_default:
             f.write(f_default.read())
 
         f.write('\n    renku_options:'
                 '\n        renku_gitlab_repository_url: "https://renkulab.io/gitlab/gabriele.barni/test-dispatcher-endpoint"'
-                f'\n        renku_gitlab_token_name: "{os.getenv("RENKU_GITLAB_TOKEN_NAME", "token_name")}"'
-                f'\n        renku_gitlab_token: "{os.getenv("RENKU_GITLAB_TOKEN", "token")}"')
+               f'\n        renku_gitlab_token_name: "{os.getenv("RENKU_GITLAB_TOKEN_NAME", "token_name")}"'
```

Comment:
that's `user name` I think

Reply:
no I need the name of the token

Reply:
what is "the name of the token", I do not understand the term. Internal token names of github? They should not be used.
It's https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html yes? And you use it in the URL as username, between schema and hostname.

Reply:
Yes it's the personal access token, that has a name assigned at creation.

What do you suggest to use instead?

Reply:
btw I see the security implications, they should not be used within URL

Reply:
I would have used ssh key here actually. Or still token but in netrc. So it's not logged.
Maybe this git library can pass the key more covertly.

I would suggest to call it username, because that's what you have to put in it's place. In practice, this username is apparently not currently used https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html . So it does not matter what would be put there.

Reply:
Now uses ssh, and no longer any token. However I needed to provide the 'renku_user_name' so to properly build the output url.